.mergify.yml

```diff
@@ -18,41 +18,41 @@ pull_request_rules:
             git merge upstream/{{base}}
             git push upstream {{head}}
             ```
-  - name: backport patches to 8.2 branch
+  - name: backport patches to 8.3 branch
     conditions:
       - merged
       - base=main
-      - label=backport-v8.2.0
+      - label=backport-v8.3.0
     actions:
       backport:
         assignees:
           - "{{ author }}"
         branches:
-          - "8.2"
+          - "8.3"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
-  - name: backport patches to 8.1 branch
+  - name: backport patches to 8.2 branch
     conditions:
       - merged
       - base=main
-      - label=backport-v8.1.0
+      - label=backport-v8.2.0
     actions:
       backport:
         assignees:
           - "{{ author }}"
         branches:
-          - "8.1"
+          - "8.2"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
-  - name: backport patches to 7.17 branch
+  - name: backport patches to 8.1 branch
```

Comment:
so `7.17` will not have any default backport, is that the expected behaviour?

Reply:
We always move the branch window one level. Although, thinking it twice, we may need to move the 8.minor pointer, not the 7 one. Therefore, we should keep:

7.17 > 8.2 > 8.3 > main

Is this correct?

Reply:
I think so, `7.17` is the last `7` minor to be supported until `9` as per https://www.elastic.co/support/eol

Reply:
I kept 7.17, removing 8.1 instead. Thanks for the clarification!